### PR TITLE
Cypress navigation update

### PR DESCRIPTION
**Pull request by maxsebso**
now includes login capabilities

### CODE REVIEW

**Comment by jackblanc** [comment]
You have merge conflicts here: 
git checkout master && git pull && git checkout [branch] && git merge master

**Comment by jackblanc** [comment]
Try merging again and running locally to see what happens.

File: tests/e2e/specs/navigationWithLogin.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
+
+
+
+describe('Testing Navigation WITH Login', () => {
```
Comment (jackblanc) [inline tests/e2e/specs/navigationWithLogin.js:4]: There's honestly really no reason to test any of the "without login" functionality, cause at this point it doesn't look like there will be anything public.

File: tests/e2e/specs/navigationWithLogin.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+    beforeEach(() => {
+        cy.login();
```
Comment (jackblanc) [inline tests/e2e/specs/navigationWithLogin.js:7]: Can you be clear about what kind of user these tests are for?

File: tests/e2e/specs/navigationWithoutLogin.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    cy.url().should('include', '/sign-up');
+
+  });
+
+  it('Visits all pages from sign up', () => {
+    cy.visit('/sign-up');
+
+    //connection to landing page
+    cy.get('.title').click();
+    cy.url().should('eq', 'http://localhost:8080/');
```
Comment (jackblanc) [inline tests/e2e/specs/navigationWithoutLogin.js:80]: What is this? Probably shouldn't be hardcoded

File: tests/e2e/support/commands.js [hunk 4]
```diff
@@ -2,6 +2,18 @@
 // This command logs you in as either a GP (General Participant), PF (Participating Family) or Admin
 // ** Right now just logs you in as an example user **
 Cypress.Commands.add("login", () => {
+    const user = {
```
Comment (jackblanc) [inline tests/e2e/support/commands.js:5]: Could you write this to be specific to the type of user, or configurable with an argument?

Comment (maxsebso) [reply tests/e2e/support/commands.js:5]: I do not have any other credentials so I can only login as PF

File: tests/e2e/specs/navigation.js [hunk 5]
```diff
@@ -1,181 +1,188 @@
...
+  const registrationsPath = '/my-events';
 
     beforeEach(() => {
         cy.login();
     });
     
     it('Test that urls take you to correct page', () => {
       //landing page
-      cy.visit('/');
+      //***** Currently failing because routing to page is wrong *****
```
Comment (jackblanc) [inline tests/e2e/specs/navigation.js:16]: This isn't actually wrong, there's no home page anymore.

**Pull request merged**